packages/@markuplint/vue-parser/src/parse.ts

```diff
@@ -187,6 +200,25 @@ function nodeize(
 				nodeName: tagName,
 				type: 'starttag',
 				namespace: originNode.namespace,
+				elementType: detectElementType(
+					tagName,
+					options?.authoredElementName,
+					/**
+					 * @see https://vuejs.org/api/built-in-components.html
+					 * @see https://vuejs.org/api/built-in-special-elements.html
+					 */
+					[
+						'component',
+						'slot',
+						'Transition',
+						'TransitionGroup',
+						'KeepAlive',
+						'Teleport',
+						'Suspense',
+						// Backward compatibility
+						/^[A-Z]/,
+					],
```

Comment:
@yamanoku 

I defined these here.
And I also defined the regex for the backward compatibility just in case.